core/src/services/iframe.js

```diff
@@ -34,11 +34,15 @@ class IframeClass {
     return container && container.length > 0 ? container[0] : undefined;
   }
 
-  getAllIframes(modalIframe) {
+  getAllIframes(iframeTypeArr = []) {
```

Comment:
Maybe we also need to update LuigiElements.getMicrofrontendIframes(). Probably we can get rid of `getAllIframes` and use also internally Luigi Core api. But then we might need to extend the elements and provide functions like: `getAllIMicrofrontendIframes`, `getMainViewMicrofrontendIframes`, `getSplitViewMicrofrontendIframes` and `getModalMicrofrontendIframes`.

Reply:
Guess we do not want to expose so many features in Core Api yet.